cdk.json

```diff
@@ -15,6 +15,16 @@
     ]
   },
   "context": {
```

Comment:
It seems like this changes how we deploy? (I'm getting an error about an invalid bucket name when I do cdk synth: `RuntimeError: Invalid S3 bucket name (value: cdk-hnb659fds-assets-<account_id>-us-west-2`). If so, the readme instructions should be updated.

Reply:
Yes. There is a template that I will message you that you put in the same directory as app.py. I will update the README.md.